app/models/ride_change_log.rb

```diff
@@ -0,0 +1,5 @@
+class RideChangeLog < ApplicationRecord
+    belongs_to :ride 
+    enum :prev_status [:canceled, :initial_pickup, :clinic_dropoff, :clinic_pickup, :final_dropoff, :completed]
+    enum :updated_status [:canceled, :initial_pickup, :clinic_dropoff, :clinic_pickup, :final_dropoff, :completed]
```

Comment:
These are the same statuses as `Ride`- let's see if we can define them in one place and re-use them!